addons/mail/static/src/models/device/device.js

```diff
@@ -20,7 +20,7 @@ registerModel({
          * Called when messaging is started.
          */
         start() {
-            // TODO FIXME Not using this.env.browser because it's proxified, and
+            // TODO FIXME Not using this.messaging.browser because it's proxified, and
```

Comment:
This comment might be obsolete and code adapted? There are occurrences of `browser.addEventListener` in the rest of the code.

Reply:
`env.browser` does not hold a reference to `addEventListener`/`removeEventListener`. I imported  the new browser (which is quiet disapointing given that the scope of this PR is to add an indirection via `messaging` but it seems better anyway than using the `window` object ) in order to use it there + deleted the comment.

Reply:
I would have left that to the PR of `env` then as indeed it's not related here. But ok now that it's done, it's fine to keep it, I was just afraid that messaging was deleted, but if it's because of listerners then fine.